src/test/java/edu/harvard/iq/dataverse/datasetutility/FileSizeCheckerTest.java

```diff
@@ -6,19 +6,45 @@
 package edu.harvard.iq.dataverse.datasetutility;
 
 import static edu.harvard.iq.dataverse.datasetutility.FileSizeChecker.bytesToHumanReadable;
+
+import edu.harvard.iq.dataverse.datasetutility.FileSizeChecker.FileSizeResponse;
 import edu.harvard.iq.dataverse.util.BundleUtil;
+import edu.harvard.iq.dataverse.util.SystemConfig;
+
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import org.junit.Test;
-import static org.junit.Assert.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /**
  *
  * @author oscardssmith
  */
 public class FileSizeCheckerTest {
 
+    private FileSizeChecker fileSizeChecker;
+
+    @BeforeEach
+    public void setUp() {
```

Comment:
I'd prefer if this override is just done in each method its used. You are already doing one method override and moving this into the other two where it is relevant will make things less confusing if anyone adds more tests.

Reply:
Okay, I moved them directly into the test cases.